python/lsst/ap/association/assoc_db_sqlite.py

```diff
@@ -377,6 +385,10 @@ def _get_dia_objects(self, indexer_indices):
         for row in self._query_dia_objects(indexer_indices):
             dia_object_record = \
                 self._dia_object_converter.source_record_from_db_row(row)
+            if expMd is not None and \
+               not self._check_dia_object_possition(dia_object_record,
```

Comment:
"position" not "possition"

Reply:
I've moved the "is not None" check into the function and changed the next lines to be within the if statement so that now only "if self._check_dia_object_position" is checked.